lib/main.js

```diff
@@ -60,7 +70,23 @@ export default {
         // Add the file to be linted
         parameters.push(filePath);
 
-        const output = await helpers.exec(executablePath, parameters, options);
+        let output;
+        try {
+          output = await helpers.exec(executablePath, parameters, options);
+        } catch (e) {
+          showErrorNotification('HAML-Lint: Unexpected error', e.message);
+          return null;
+        }
+
+        if (output.exitCode !== 0) {
```

Comment:
Is there an easy method to trigger this so it can be added to the specs?

Reply:
I think using a ruby version with an old compilant syntax.

Reply:
Oh! This is the provider that had that, it's super easy to trigger when the CI version of Ruby is no longer supported by HAML, used to happen all the time.

That being said... the only way to test that would be a dedicated CI job so this is probably more complex than it's worth to implement it, I'm fine with leaving it without specs for now since the rest of the code isn't 100% tested in CI anyway.

Reply:
Ok, we have time to think in how to do a spec.

Reply:
@Arcanemagus  - on my setup when any lints don't pass, the haml_lint returns with an exit code of 65, so this always triggers.  This is what's causing #169.  Are you familiar with what might be causing this?